recipe/meta.yaml

```diff
@@ -1,41 +1,72 @@
-{% set version = "2020.1" %}
+{% set version = "2020a" %}
```

Comment:
Although `conda` also uses Python-like versioning, `tzdata` uses consistent versions for which the non-numeric suffix is always lexicographical increasing.
Meaning: It's fine for us to use the upstream version scheme.